### ISSUE TITLE
Keyboard focused items are not visible

### ISSUE THREAD

**Issue by furiousdom** [Hyzeta/vscode-theme-github-light#14]
### Main Flow

Location: Activity Bar -> Search

Flow: When keyboard cursor is blinking in the Search field, press "Tab" on the keyboard.

One thing to note is that if I hover over the Match Case button with a mouse the focus indicator appears.

### Current behaviour

Match Case button doesn't have a focus indicator.

### Expected behaviour

Match Case button should have a focus indicator.
